### ISSUE TITLE
b-nav-form accessibility issue

### ISSUE THREAD

**Issue by adholland** [bootstrap-vue-next/bootstrap-vue-next#1346]
package: 0.9.16

Using a b-nav-form within a b-nav creates an accessibility failure : -

ul and ol must only directly contain li, script or template elements
serious
wcag2 awcag131

Looks like it should be a simple fix whereby the b-nav-form should contain, at its root,  an li element.

I can fix the issue locally by wrapping the b-nav-form in a li element: -
```
<b-nav>
        <b-nav-item
                   ...
                >
                    text
        </b-nav-item>
        <li>
            <b-nav-form class="basket-avatar">
                <Basket
                   ...
                />
            </b-nav-form>
        </li>
</b-nav>
```

**Comment by adholland** [comment]
@VividLemon any chance of pushing 0.10.2 to npm ? Could really do with it.
